src/lobby.cpp

```diff
@@ -441,8 +441,9 @@ void Lobby::on_server_list_clicked(QTreeWidgetItem *p_item, int column)
 // doubleclicked on an item in the serverlist so we'll connect right away
 void Lobby::on_server_list_doubleclicked(QTreeWidgetItem *p_item, int column)
 {
+  doubleclicked = true;
```

Comment:
Can this never be reset to `false`?

Reply:
i had assumed the lobby object was destroyed when connecting to a server. given ao2 jank, i guess that isn't the case then...

Reply:
regardless, the "doubleclicked = true" codepath doesn't seem to execute if you join a server and then return back to the lobby